win32/src/pe/resources.rs

```diff
@@ -56,12 +56,31 @@ pub enum RT {
     STRING = 6,
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, Eq)]
 pub enum ResourceName<'a> {
     Name(&'a Str16),
     Id(u32),
 }
 
+impl<'a> PartialEq for ResourceName<'a> {
+    // Case insensitive comparison is used for string names
+    fn eq(&self, other: &Self) -> bool {
+        match self {
+            Self::Name(name) => match other {
```

Comment:
It doesn't matter at all, but you could write this

match (self, other) {
  (Self::Name(...), Self::Name(...)) => ...
  etc
}

Reply:
Thanks for the tip!